yt_dlp/YoutubeDL.py

```diff
@@ -2720,8 +2721,8 @@ def to_screen(*msg):
                 offset, duration = info_dict.get('section_start') or 0, info_dict.get('duration') or float('inf')
                 if chapter or offset:
                     new_info.update({
-                        'section_start': offset + chapter.get('start_time', 0),
-                        'section_end': offset + min(chapter.get('end_time', duration), duration),
+                        'section_start': offset + chapter.get('start_time', 0) if chapter.get('start_time') is not None else None,
+                        'section_end': offset + min(chapter.get('end_time', duration), duration) if chapter.get('end_time') is not None else None,
```

Comment:
Wouldn't this be much simpler if we parse empty range to `0` and `float('inf')`? That would also simplify the above printing

Reply:
That way it complicates FFmpegFD (especially the duration calculation for `-t`)